### PR TITLE
Adaptation for AUV7

### PR DESCRIPTION
**Pull request by guilsch**
## Description
adapt rs485Msg slave for AUV7 and AUV8.

## How has this been tested ?
Tested on docker with rostopic echo /interface_rs485/dataRx

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/provider_thruster/provider_thruster_node.cc [hunk 1]
```diff
@@ -24,6 +24,20 @@ namespace provider_thruster {
...
+            SLAVE = sonia_common::SendRS485Msg::SLAVE_ESC;    
+        }
+
+        else if (strcmp(auv, "AUV7") == 0){
+            SLAVE = sonia_common::SendRS485Msg::SLAVE_PWR_MANAGEMENT;
+        }
+
+        else if (strcmp(auv, "LOCAL") == 0){
+            SLAVE = sonia_common::SendRS485Msg::SLAVE_PWR_MANAGEMENT;
+        }
```
Comment (supertoto29) [inline src/provider_thruster/provider_thruster_node.cc:40]: I would add a default so if no variable are there we fall back on something. See suggestion
```suggestion
        }
        else
        {
        ROS_ERROR_STREAM("No environnement variable found, using the default from AUV8");
        SALVE = sonia_common::SendRS485Msg::SLAVE_PWR_MANAGEMENT;
        }
```

**Review by supertoto29 — CHANGES_REQUESTED** [review]

File: src/provider_thruster/provider_thruster_node.cc (hunk 1 above)
Comment (guilsch) [reply src/provider_thruster/provider_thruster_node.cc:40]: if no variable slave = SLAVE_ESC for auv8 not SLAVE_POWER_MANAGEMENT am I right ?

**Review by supertoto29 — APPROVED** [review]
LGTM!

**Comment by supertoto29** [comment]
@guilsch  I will need to rename the branch

**Pull request closed without merging**